packages/bootstrap/scss/scrollview/_theme.scss

```
@@ -2,34 +2,89 @@
 
     // refactor when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
     kendo-scrollview.k-scrollview-wrap,
-    div.k-scrollview {
+    .k-scrollview {
         @extend .k-text-selection !optional;
         @include fill(
             $scrollview-text,
             $scrollview-bg,
             $scrollview-border
         );
 
-        .k-scrollview-pageable,
-        .k-scrollview-nav {
-            margin-left: $carousel-control-width;
-            margin-right: $carousel-control-width;
-            width: auto;
-            left: 0;
-            right: 0;
-
-            > li {
-                background-color: $scrollview-pagebutton-bg;
-                margin-right: $carousel-indicator-spacer;
-                margin-left: $carousel-indicator-spacer;
-                padding: 0;
-                max-width: $carousel-indicator-width;
-                height: $carousel-indicator-height;
-                border-width: 0;
-                border-radius: 0;
-                flex: 1 0 auto;
-                position: relative;
+        &.k-scrollview-dark {
+            kendo-scrollview-pager,
+            .k-scrollview-nav-wrap {
+                background-color: $scrollview-dark-bg;
             }
+            .k-scrollview-next,
+            .k-scrollview-prev {
+                color: $scrollview-dark-bg;
+
+            }
+        }
+
+        &.k-scrollview-light {
+            kendo-scrollview-pager,
+            .k-scrollview-nav-wrap {
+                background-color: $scrollview-light-bg;
+            }
+
+        }
+    }
+
+    kendo-scrollview.k-scrollview-wrap .k-scrollview,
+    .k-widget.k-scrollview .k-scrollview-wrap {
+        &:focus {
+            outline: none;
+
+
+        }
+    }
```

```suggestion
    .k-scrollview,
    .k-scrollview-wrap {
        outline: none;
    }
```

or if that doesn't work 

```suggestion
    .k-scrollview:focus,
    .k-scrollview-wrap:focus {
        outline: none;
    }
```